lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -91,18 +91,28 @@ final String canvasKitBuildUrl =
     canvasKitBaseUrl + (kProfileMode ? 'profiling/' : '');
 final String canvasKitJavaScriptBindingsUrl =
     canvasKitBuildUrl + 'canvaskit.js';
-String canvasKitWasmModuleUrl(String file) => canvasKitBuildUrl + file;
+String canvasKitWasmModuleUrl(String file) => _currentCanvasKitBase! + file;
+
+/// The script element which CanvasKit is loaded from.
+html.ScriptElement? _canvasKitScript;
+
+/// A [Future] which completes when the CanvasKit script has been loaded.
+Future<void>? _canvasKitLoaded;
+
+/// The currently used base URL for loading CanvasKit.
+String? _currentCanvasKitBase;
 
 /// Initialize CanvasKit.
 ///
 /// This calls `CanvasKitInit` and assigns the global [canvasKit] object.
-Future<void> initializeCanvasKit() {
+Future<void> initializeCanvasKit({String? canvasKitBase}) {
```

Comment:
Where to we get and pass the value of `canvasKitBase`?

Reply:
I was thinking either pass it in `setUpCanvasKitTests` or read it from a `<meta>` tag in index.html